packages/react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -708,7 +709,9 @@ function useMouseHandlers(
 export const ContextualMenuBase: React.FunctionComponent<IContextualMenuProps> = React.memo(
   React.forwardRef<HTMLDivElement, IContextualMenuProps>((propsWithoutDefaults, forwardedRef) => {
     const { ref, ...props } = getPropsWithDefaults(DEFAULT_PROPS, propsWithoutDefaults);
-    const hostElement = React.useRef<HTMLDivElement>(null);
+    const rootRef = React.useRef<HTMLDivElement>(null);
+    const hostElement = useMergedRefs(forwardedRef, rootRef);
```

Comment:
`forwardedRef` is used as the ref for the `<Callout>`, while `hostElement` is used as the ref for the child menu of the Callout. I'm not sure why those are the way they are, but I don't think they should be merged 😅

Reply:
oooooh, good catch sarah. i'll double check where that ref is getting placed.